backend/src/Models/ElectionRolls.ts

```diff
@@ -106,13 +108,13 @@ export default class ElectionRollDB {
 
     update(election_roll: ElectionRoll, ctx:ILoggingContext, reason:string): Promise<ElectionRoll | null> {
         Logger.debug(ctx, `ElectionRollDB.updateRoll`);
-        var sqlString = `UPDATE ${this._tableName} SET ballot_id=$1, submitted=$2, state=$3, history=$4  WHERE election_id = $5 AND voter_id=$6`;
+        var sqlString = `UPDATE ${this._tableName} SET ballot_id=$1, submitted=$2, state=$3, history=$4, registration=$5 WHERE election_id = $6 AND voter_id=$7`;
```

Comment:
This is outside the scope of this PR, but we should probably think about how to handle concurrent modifications down the line

Let's say there are currently 3 events in the history, and 2 clients try adding a 4th event at the same time. If the flow happened as follows, the second client could overwrite the addition on the first client added

Example
```
Client1: Downloads history (w/ 3 elements) 
Client2: Downloads history (same 3 elements)
Client1: Locally adds 4th event
Client2: Locally adds 4th event
Client1: Uploads their modified history
Client2: Uploads their modified history (thus overwriting client1's addition)
```

I've used optimistic locking ([explainer vid](https://www.youtube.com/watch?v=VxGKvqHhU5c)) for this in the past, but we can discuss other options